src/components/Releases/ReleaseOverviewTab.tsx

```diff
@@ -34,22 +34,22 @@ const ReleaseOverviewTab: React.FC = () => {
   const { releaseName } = useParams<RouterParams>();
   const namespace = useNamespace();
   const [release] = useRelease(namespace, releaseName);
-  const [managedPrNamespace, managedPipelineRun] = getNamespaceAndPRName(
-    getManagedPipelineRunFromRelease(release),
-  );
-  const [tenantPrNamespace, tenantPipelineRun] = getNamespaceAndPRName(
-    getTenantPipelineRunFromRelease(release),
-  );
-  const [tenantCollectorPrNamespace, tenantCollectorPipelineRun] = getNamespaceAndPRName(
-    getTenantCollectorPipelineRunFromRelease(release),
-  );
-  const [finalPrNamespace, finalPipelineRun] = getNamespaceAndPRName(
-    getFinalPipelineRunFromRelease(release),
-  );
-  const [releasePlan, releasePlanLoaded] = useReleasePlan(namespace, release.spec.releasePlan);
+  const [managedPrNamespace, managedPipelineRun] = release
+    ? getNamespaceAndPRName(getManagedPipelineRunFromRelease(release))
```

Comment:
nit: maybe we could update `getManagedPipelineRunFromRelease` directly to handle nullable/undefined `release`? so we don't need to be updating these lines where we're using the function :)